charts/gluu/values.yaml

```diff
@@ -363,6 +363,16 @@ config:
     cnSqlDbTimezone: UTC
     # -- SQL password  injected the secrets .
     cnSqldbUserPassword: Test1234#
+    # -- Enforce connection to SQL database using SSL.
+    cnSqlSslEnabled: false
+    # -- Mode when connecting to SQL database using SSL. If using MySQL, choose one of `PREFERRED`, `REQUIRED`, `VERIFY_CA`, or `VERIFY_IDENTITY`. If using PostgreSQL, choose one of `allow`, `prefer`, `require`, `verify-ca`, or `verify-full`.
+    cnSqlSslMode: ""
+    # -- Base64-encoded string of CA certificate used to sign client/server certificate of MySQL/PostgreSQL server. Required if using client cert authentication.
+    cnSqlSslCaCert: ""
+    # -- Base64-encoded string of client certificate signed by CA. Required if using client cert authentication.
+    cnSqlSslClientCert: ""
+    # -- Base64-encoded string of client key signed by CA. Required if using client cert authentication.
+    cnSqlSslClientKey: ""
```

Comment:
Same comment wording issue; tighten phrasing for the private key.


Apply this edit:

```diff
-    # -- Base64-encoded string of client key signed by CA. Required if using client cert authentication.
+    # -- Base64-encoded client private key corresponding to the client certificate. Required if using client cert authentication.
```